opentelemetry-sdk/src/opentelemetry/sdk/metrics/_internal/exponential/aggregation.py

```diff
@@ -0,0 +1,555 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from math import inf
+from typing import Optional, TypeVar, Union
+
+from opentelemetry.sdk.metrics._internal.aggregation import (
+    AggregationTemporality,
+    _Aggregation,
+)
+from opentelemetry.sdk.metrics._internal.exponential.buckets import Buckets
+from opentelemetry.sdk.metrics._internal.exponential.config import (
+    DEFAULT_MAX_SIZE,
+    MAX_MAX_SIZE,
+    MIN_MAX_SIZE,
+)
+from opentelemetry.sdk.metrics._internal.exponential.exponent import (
+    ExponentMapping,
+)
+from opentelemetry.sdk.metrics._internal.exponential.logarithm_mapping import (
+    LogarithmExponentialHistogramMapping,
+)
+from opentelemetry.sdk.metrics._internal.measurement import Measurement
+from opentelemetry.sdk.metrics._internal.point import Buckets as BucketsPoint
+from opentelemetry.sdk.metrics._internal.point import (
+    ExponentialHistogramDataPoint,
+)
+from opentelemetry.sdk.metrics._internal.point import (
+    Histogram as HistogramPoint,
+)
+from opentelemetry.sdk.metrics._internal.point import (
+    HistogramDataPoint,
+    NumberDataPoint,
```

Comment:
(Why's there a NumberDataPoint in this file?)

Reply:
It is used to define a type [here](https://github.com/open-telemetry/opentelemetry-python/pull/2930/files/6c85b308293d5fb837fda2a22993411b0507ad54#diff-022180f24f5237c3ae4d43d3c7a09b9196f39a1b94af6b25a9a9da860b807153R48).